README.md

```diff
@@ -43,24 +54,179 @@ The plugin will remove the custom attribute from the element and replace its con
 You can use external libraries for this as well, no problem. Just make sure you are passing in a function that takes a string and returns a string. You might have to wrap the library function if it doesn't behave like this, but it will work with anything that transforms content.
 
 ```html
-<p md>Wow, it's **markdown**!</p>
+<p key>Transform me!</p>
```

Comment:
What would be a better generic name in your opinion, if any ? I tried to provide a generic example